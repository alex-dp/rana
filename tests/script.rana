html() {
	head(){
		title(){"Demo of a script in Rana"}
	}

	body() {
		script() {
			"
			var a = \"this demo is working!!\"
			alert(a)
			"
		}
	}
}