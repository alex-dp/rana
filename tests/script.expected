<html> 
	<head>
		<title>Demo of a script in Rana</title>
	</head>

	<body> 
		<script> 
			
			var a = "this demo is working!!"
			alert(a)
			
		</script>
	</body>
</html>